dspace-api/src/main/java/org/dspace/content/Community.java

```diff
@@ -46,7 +46,7 @@
     )
     private final List<Community> subCommunities = new ArrayList<>();
 
-    @ManyToMany(fetch = FetchType.LAZY, mappedBy = "subCommunities")
+    @ManyToMany(fetch = FetchType.EAGER, mappedBy = "subCommunities")
```

Comment:
Is that necessary? I'm afraid this could create problems for repositories with a lot of communities.